package.json

```diff
@@ -5,7 +5,11 @@
     "i18next-parser": "^0.12.0",
     "lerna": "2.5.1",
     "npm-watch": "^0.1.8",
-    "surge": "^0.18.0"
+    "pixelmatch": "^4.0.2",
+    "pngjs": "^3.3.2",
+    "puppeteer": "^1.2.0",
+    "surge": "^0.18.0",
```

Comment:
Why is it needed?
Travis installs it as global

Reply:
Surge is installed locally for our releases. It allows us not to install it globally on our computers, but use npm scripts to publish the demo site.

Reply:
Why not ?
If not it means to use the script I need to do some global install.